panther_analysis_tool/backend/lambda_client.py

```diff
@@ -32,7 +31,6 @@
     BackendResponse,
     BulkUploadParams,
     BulkUploadResponse,
-    BulkUploadStatistics,
```

Comment:
Isn't this still used for regular bulk upload? Is that just not in this file?

Reply:
Good catch. I'll add that back

Reply:
Oh wait nevermind it was just no longer being used in this file. That class is still there